hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1072,6 +1074,19 @@ message KeyInfo {
     optional bool isFile = 19;
 }
 
+message BasicKeyInfo {
+    required string volumeName = 1;
+    required string bucketName = 2;
```

Comment:
We can skip this; it is part of the request and can be part of the response only once.

Reply:
Thank you for the review, I have removed both the fields from `BasicKeyInfo` and hence the `ListKeysLightResponse`.